### DISCUSSION TITLE
LTI Support

### DISCUSSION THREAD

**Discussion by colabcolibri** [stoatchat/discussions#676] [Feature Suggestions]
Hey all,

I just found Revolt, while looking for a chat platform to use with Moodle.

I would like to know if you have support to LTI (Learning Tools Interoperability).

It would be awesome, once I could set up a server and run direct inside Moodle, or other LMS.

**Comment by IAmTomahawkx** [comment]
We do not, no. We're missing several components required for something like this, and as of right now our focus is on other things, such as mobile apps.
